packages/block-library/src/latest-posts/index.php

```diff
@@ -150,9 +150,15 @@ function render_block_core_latest_posts( $attributes ) {
 			$list_items_markup .= sprintf(
 				'<div class="wp-block-latest-posts__post-excerpt">%1$s</div>',
 				$trimmed_excerpt
+				
+			);
+			$list_items_markup .= sprintf(
+				'<a class="wp-block-latest-posts__post-read-more" href="%1$s">%2$s</a>',
```

Comment:
This link also needs an `aria-label` with the post title.

Reply:
Added